src/term/mod.rs

```diff
@@ -1944,7 +1944,7 @@ impl ansi::Handler for Term {
     /// set a terminal attribute
     #[inline]
     fn terminal_attribute(&mut self, attr: Attr) {
-        trace!("Set Attribute: {:?}", attr);
+        trace!("set_attribute: {:?}", attr);
```

Comment:
Why the underscore here? Makes it look like `set_attribute` is a method or variable here. Maybe `setting attribute:` would be better?